content/guide/06-special-components.md

```diff
@@ -50,25 +50,25 @@ If you don't know what kind of component to render until the app runs — in ot
 
 ```html
 <!--{ hidden: true, filename: 'Red.html' }-->
-<p style='color: red'>Red {{name}}</p>
+<p style="color: red">Red {name}</p>
 ```
 
 ```html
 <!--{ hidden: true, filename: 'Blue.html' }-->
-<p style='color: blue'>Blue {{name}}</p>
+<p style="color: blue">Blue {name}</p>
 ```
 
 > Note that `Red` and `Blue` are items in `data`, *not* `components`, unlike if we were doing `<Red>` or `<Blue>`.
 
 The expression inside the `{...}` can be any valid JavaScript expression. For example, it could be a [computed property](guide#computed-properties):
```

Comment:
Needs to be updated for the new syntax.

Reply:
The "The expression inside the `{...}`" bit. I mean, yeah, the expression is still in braces, but I think it should talk about the `this=` part of the syntax somewhere other than in the example.